core/src/CMakeLists.txt

```diff
@@ -57,9 +60,9 @@ set(_srcs
 
 set(_private_headers
   util/usAtomicInt_p.h
+  util/usFrameworkPrivate_p.h
```

Comment:
Please rename to `usFrameworkPrivate.h`. The `_p` is only used if there is no `Private` suffix in the name already.